CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - An issue where the taker logs were spammed with unnecessary error message on auto-rollover.
 - An issue where metrics on connected peers where not reported correctly.
 
+### Changed
+
+- Breaking change: with bdk>0.20 key derivations will take into account the network, i.e. for mainnet it uses `m/84'/0'/0'` and for testnet/regtest it uses `m/84'/1'/0'`. This was caused by https://github.com/bitcoindevkit/bdk/pull/585.
+
```

Comment:
maybe we can expand in lay terms that users with testnet wallets should move their funds off ItchySats wallet before upgrading or return them to the faucet.